### PR TITLE
Use hostname of server.address / management.address if set and prefer-ip is false

### PR DESCRIPTION
**Pull request by weeniearms**
Fix for #295

### CODE REVIEW

**Comment by weeniearms** [comment]
The decrease of coverage has to do with the fact that certain lines of code have been removed. No new lines that are not covered with tests have been introduced.

File: spring-boot-admin-starter-client/src/main/java/de/codecentric/boot/admin/config/AdminClientProperties.java [hunk 1]
```diff
@@ -134,10 +134,7 @@ public String getServiceUrl() {
 		}
 
 		if (preferIp) {
-			InetAddress address = server.getAddress();
-			if (address == null) {
-				address = getHostAddress();
-			}
+			InetAddress address = getHostAddress();
```
Comment (joshiste) [inline spring-boot-admin-starter-client/src/main/java/de/codecentric/boot/admin/config/AdminClientProperties.java]: Please move the local variable address out of the if-block and get rid of the getHostAddress() call in Line 145

**Comment by joshiste** [comment]
merged into master  and 1.3.x with e8d80584c3f5ab26dff9c345d32f1be8c97c6cf1

**Comment by joshiste** [comment]
1.3.7 is released.

**Pull request closed without merging**